docs/data/material/components/transitions/SimpleSlide.tsx.preview

```diff
@@ -3,7 +3,9 @@
     control={<Switch checked={checked} onChange={handleChange} />}
     label="Show"
   />
-  <Slide direction="up" in={checked} mountOnEnter unmountOnExit>
-    {icon}
-  </Slide>
+  <Box sx={{ overflow: 'hidden' }}>
```

Comment:
Is this Box still needed?

Reply:
No @michaldudak

Reply:
HI @michaldudak My checks are failing multiple time, what should i do?

Reply:
It seems you did not commit SimpleSlide.tsx.preview after running the formatting commands.

Reply:
Oh got it, Thanks you @michaldudak